### PR TITLE
ENH: Adds alert persistent feature

### LINKED ISSUE #35: scaling to 0 removes alert rules
I am trying to get the benefits of serverless, with microservices by autoscaling based on queue length. When there are no messages in the queue, the service can be scaled down to 0.

I'm using @thomasjpfan's https://github.com/thomasjpfan/docker-scaler, though Jenkins job would be the same, and a rabbitmq prometheus exporter. This allows me to scale without instrumenting the service specifically.

```
~ $ docker service logs voelhgexsur4
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:21:32 Scale service down: rethink-denormalizer_service
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:21:32 scale-service success: Scaling rethink-denormalizer_service from 1 to 0 replicas (min: 0)
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:21:32 Alertmanager received message: Scaling rethink-denormalizer_service from 1 to 0 replicas (min: 0)
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:31:32 Scale service down: rethink-denormalizer_service
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:31:32 scale-service success: Scaling rethink-denormalizer_service from 1 to 0 replicas (min: 0)
monitor_scaler.1.voelhgexsur4@ip-172-31-0-9.ec2.internal    | 2018/02/20 07:31:32 Alertmanager received message: Scaling rethink-denormalizer_service from 1 to 0 replicas (min: 0)
```

It works great.

Problem is, this bring the service down in the eyes of DFM, and correctly, for DFP.

In DFM this is an issue because now, because the alerts needed to scale back up when there are messages in the queue again have been removed as part of the service removed event.

### PR DESCRIPTION
**Pull request by thomasjpfan**
Closes #35

### CODE REVIEW

**Comment by vfarcic** [comment]
It's available in the [release 18.02.21-52](https://github.com/vfarcic/docker-flow-monitor/releases/tag/18.02.21-52)

File: server/server.go [hunk 1]
```diff
@@ -241,6 +241,8 @@ func (s *serve) getAlerts(req *http.Request) []prometheus.Alert {
 		alertName := req.URL.Query().Get(fmt.Sprintf("alertName.%d", i))
 		annotations := s.getMapFromString(req.URL.Query().Get(fmt.Sprintf("alertAnnotations.%d", i)))
 		labels := s.getMapFromString(req.URL.Query().Get(fmt.Sprintf("alertLabels.%d", i)))
+		persistant := req.URL.Query().Get(fmt.Sprintf("alertPersistent.%d", i)) == "true"
```
Comment (patrickleet) [inline server/server.go]: not a big deal functionally, but there is a typo here:

`persistant` should be `persistent`

Comment (thomasjpfan) [reply server/server.go]: Thanks! PR in https://github.com/vfarcic/docker-flow-monitor/pull/37

**Comment by patrickleet** [comment]
Sweet - tried this out already - works great! Thanks for the fast PR! Much appreciated!

**Pull request merged**